doc/harmony_ap_clustering_user_guide.md.md

```diff
@@ -0,0 +1,496 @@
+            <redeliveryPlugin fallbackToDeadLetter="true"
+                              sendToDlqIfMaxRetriesExceeded="true">
+                <redeliveryPolicyMap>
+                    <redeliveryPolicyMap>
+                        <defaultEntry>
+                            <!-- default policy-->
+                            <redeliveryPolicy maximumRedeliveries="10" redeliveryDelay="300000"/>
+                        </defaultEntry>
+                        <redeliveryPolicyEntries>
+                            <redeliveryPolicy queue="domibus.internal.retentionMessage.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.dispatch.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.largeMessage.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.splitAndJoin.queue" maximumRedeliveries="3"/>
+                            <redeliveryPolicy queue="domibus.internal.pull.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.pull.receipt.queue" maximumRedeliveries="3"/>
+                            <redeliveryPolicy queue="domibus.internal.alert.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.earchive.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.internal.earchive.notification.queue" maximumRedeliveries="6" redeliveryDelay="1800000"/>
+                            <redeliveryPolicy queue="domibus.internal.earchive.notification.dlq" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.fsplugin.send.queue" maximumRedeliveries="0"/>
+                            <redeliveryPolicy queue="domibus.wsplugin.send.queue" maximumRedeliveries="0"/>
+                        </redeliveryPolicyEntries>
+                    </redeliveryPolicyMap>
+                </redeliveryPolicyMap>
+            </redeliveryPlugin>
+
+            <discardingDLQBrokerPlugin dropAll="false" dropOnly="domibus.internal.earchive.notification.dlq domibus.internal.dispatch.queue domibus.internal.pull.queue domibus.internal.alert.queue domibus.internal.earchive.queue domibus.internal.largeMessage.queue domibus.internal.retentionMessage.queue domibus.fsplugin.send.queue domibus.wsplugin.send.queue" reportInterval="10000"/>
+        </plugins>
+    </broker>
+
+</spring:beans>
+```
+
+### 3.3 Nginx configuration
+
+Nginx is used as a load balancer. The most basic configuration for the `nginx.conf` configuration file looks like this:
+
+```nginx
+events { worker_connections 1024; }
+
+http {
+    upstream harmony-ap {
+        server harmony-ap-main:8080;
+        server harmony-ap-replica:8080;
+    }
+
+    server {
+        listen 80;
+
+        location / {
+            proxy_pass http://harmony-ap/;
+        }
+    }
+}
+```
```

Comment:
It would be nice to have one more section that explains step-by-step how to deploy the example. For example:

1. Create a file `docker-compose.yml` with the example configuration in section 3.1.
2. Create two configuration files `activemq-main.xml` and `activemq-replica.xml` with the example configuration in section 3.2.
3. Create a configuration file `nginx.conf` with the example configuration in section 3.3.
4. Deploy the example by running `docker compose up`.
5. Explain how to access the AP UI...

Reply:
Like it, will add the steps by steps instructions